Khan/perseus
# [Interactive Graph] Add example function area for locked function settings

## Summary:
Having a list of sample functions available to quickly copy/paste into the locked function settings can be very helpful for content creators. This PR adds that capability, along with an updatable file (locked-function-examples.ts) for adding/modifying example functions.


Issue: LEMS-1948

## Test plan:

1. Launch Storybook (yarn start)
2. Navigate to Perseus Editor => Editor => [Demo Interactive Graph](http://localhost:6006/?path=/story/perseuseditor-editor--demo-interactive-graph)
3. Open the "interactive-graph 1" widget settings
4. Click on the "Add locked figure" drop-down at the bottom of the editor and choose "Function"
5. Below the "domain min/max" fields there is an "examples" dropdown, and instructions to the right.
6. Choose an example category from the dropdown.
   * Note that the instructions are replaced by examples for that category.
   * Currently, there are only 2 examples per category, but if you edit the DOM and add another list item, you can see that the area's height is limited, and you can scroll to see the additional example(s).
7. Copy one of the examples by clicking on the copy icon to the left of the example that you want.
8. Paste the contents of the clipboard into the equation field (above the domain min/max section).

## Affected UI
### New settings feature
![Example Equations - Updated](https://github.com/user-attachments/assets/473382e8-72eb-443d-aa61-4585f8da96c9)
